config/foreman-proxy-content-answers.yaml

```diff
@@ -14,6 +14,7 @@ certs:
 foreman_proxy_content:
   pulpcore_mirror: true
   puppet: false
+  pulpcore_cache_enabled: true
```

Comment:
This should not be needed once the PR in in puppet-foreman_proxy_content is merged. Right now it's still detecting the built in default of `false`. I'd prefer to keep this list minimal and only those where they do not match the defaults.

Reply:
ahh!  sounds good, i'll remove it

Reply:
I think I found the reason it was needed. You should write the migration slightly more specific.